### ISSUE TITLE
Parameters for 8-channel mixtures

### ISSUE THREAD

**Issue by quancs** [hangtingchen/Beam-Guided-TasNet#7]
Hello, I am training your model on a [16k 8-channel dataset](https://github.com/Audio-WestlakeU/NBSS) using the parameters for 16k given in https://github.com/hangtingchen/Beam-Guided-TasNet/issues/4. 
It's strange that on the validation set the 8-channel experiments performs much worse than the 4-channel and 2-channel ones.
So, is it possible? Or should I also change other parameters as well?

Below are the val/loss curves, where the black and purple lines are 8-channel versions, the green is the 4-channel version and the blue is the 2-channel version.
![图片](https://user-images.githubusercontent.com/16813683/188283505-4c8205c2-aded-48a0-9fd8-647e37213cd1.png)

The negative of si_sdr on validation set on reference channel: 
![图片](https://user-images.githubusercontent.com/16813683/188283598-a72137eb-d01a-47e9-b314-7809662cefe1.png)

Thanks in advance!

**Comment by quancs** [comment]
One more question is that should I change the STFT parameters specified in the MVDR module?

**Comment by hangtingchen** [comment]
sorry for the late reply. It is strange, and I cannot imply the reason directly. Maybe we can exchange comments on wechat. My account is cht465244243.
